### ISSUE TITLE
May I translate this book into Korean?

### ISSUE THREAD

**Issue by kciter** [munificent/craftinginterpreters#937]
Hello!

I'm software developer in Korea.

Here is my GitHub and Korean blog link.
https://github.com/kciter
https://kciter.so/

I'm very interested in programming languages and I was excited to read your book.

So, I would like to translate your great book into Korean with my colleagues for Korean developers.

May I translate this book into Korean?

I look forward to your reply.

**Comment by Gesrua** [comment]
A Chinese developer is translating the book in [GitHub](https://github.com/confucianzuoyuan/craftinginterpreters), and another Chinese developer is doing that on Zhihu(another Chinese forum). 

I think is a good choice to build offical translation repository, an example is [JavaScript Tutorial](https://github.com/javascript-tutorial/). It can avoid some repetitive translation work.

**Comment by heinthanth** [comment]
Yes! Me too. I would like to translate this book to Myanmar ( Burmese ). This would help developers from Myanmar!
